Python/Product/Analysis/scrape_module.py

```diff
@@ -398,6 +398,16 @@ def _init_restype_fromdocstring(self):
 
         index = first_line.index('->')
         typeName = first_line[index + 2:].strip()
+        
+        if typeName.startswith('new'):
+            typeName = typeName[3:].strip();
+        elif typeName.startswith('Return a'):
+            typeName = typeName[8:].strip();
+        elif typeName.startswith('Return'):
+            typeName = typeName[6:].strip();
+        elif typeName.startswith('Current'):
+            typeName = typeName[7:].strip();
```

Comment:
I'm not comfortable with this block - we are basically guaranteed to generate unparseable code as a result.

If there are builtin/standard library types that need help, we should special-case them explicitly in the existing tables (in this file). For third-party modules we should either special case them the same way or come up with more robust parsing to ensure that we aren't just dropping rubbish into the generated pyi.

Reply:
OK, it is minority of cases, I can take them out. Typeshed will improve this anyway.